composer.json

```diff
@@ -66,5 +66,11 @@
     "bin": ["bin/doctrine"],
     "archive": {
         "exclude": ["!vendor", "tests", "*phpunit.xml", "build.xml", "build.properties", "composer.phar", "vendor/satooshi", "lib/vendor", "*.swp"]
+    },
+    "repositories": {
+        "greg0ire": {
+            "type": "vcs",
+            "url": "https://github.com/greg0ire/dbal"
```

Comment:
```suggestion
            "type": "git",
            "url": "git@github.com:greg0ire/dbal.git"
```

Reply:
Oh right, obviously! Thanks @derrabus !

Reply:
Or maybe that wasn't the correct way after all…